src/test/Nodes1DProvisionerTests.cpp

```diff
@@ -281,7 +281,7 @@ namespace blitzdg {
 				delete nodes1DProvisioner;
 			}
 
-			It(Should_Build_4th_Order_GradVandermonde_Matrix) {
```

Comment:
Why kill this test?

Reply:
We made the member function BuildGradVandermonde private since it doesn't need to be part of the public interface as far as I can tell.

Reply:
Fair enough